File: Programming Language(ver.2)/Python/303.py
def compute(n):
    isprime = 1
    for i in range(2, int(n/2) + 1):
        if n % i == 0:
            isprime = 0
    return isprime

n = int(input())
if compute(n):
    print(str(n) + ' is a prime number')
else:
    print(str(n) + ' is not a prime number')

File: Programming Language(ver.2)/C++/303.cpp
#include <iostream>
using namespace std;

int compute(int);
int main()
{
    int n;
    cin >> n;
    if (compute(n) == 1)
        cout << n << " is a prime number";
    else
        cout << n << " is not a prime number";
    return (0);
}

int compute(int n)
{
    int i;
    for (i = 2; i * i <= n; i++) {
        if (i == n)
            return 1;
        else if (n % i == 0)
            return 0;
    }
    return 1;
}